src/Model/Template.php

```diff
@@ -55,18 +76,20 @@ public function getContainers()
      * The meta array is an array containing the
      *    - area name.
      *
-     * @param string $code
-     * @param array  $meta
+     * @param string               $code
+     * @param array<string, mixed> $meta
      */
     public function addContainer($code, $meta): void
     {
         $this->containers[$code] = $this->normalize($meta);
     }
 
     /**
-     * @param $code
+     * @param string $code
+     *
+     * @return array<string, mixed>
      *
-     * @return array
+     * @phpstan-return Container|array{}
```

Comment:
Seems weird to return `array{}` if no container was found.

It should be a LogicException (or at least null).

Reply:
I went for null instead of exceptions

Reply:
It makes sens since there is no has method